io.github.brunoanc.eternalmodmanager.json

```diff
@@ -0,0 +1,77 @@
+{
+  "id" : "io.github.brunoanc.eternalmodmanager",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "46",
+  "sdk" : "org.gnome.Sdk",
+  "sdk-extensions" : [
+    "org.freedesktop.Sdk.Extension.rust-stable"
+  ],
+  "command" : "eternal_mod_manager",
+  "separate-locales" : false,
+  "finish-args" : [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--share=network",
+    "--device=dri"
+  ],
+  "build-options" : {
+    "append-path" : "/usr/lib/sdk/rust-stable/bin",
+    "env" : {
+        "CARGO_HOME" : "/run/build/eternalmodmanager/cargo"
+    }
+  },
+  "modules" : [
+    {
+      "name" : "eternalmodmanager",
+      "buildsystem" : "simple",
+      "sources" : [
+        {
+          "type" : "git",
+          "url" : "https://github.com/brunoanc/EternalModManager.git",
+          "tag" : "v4.2.0",
+          "commit" : "039435757bd2c2089567805d1eed4a3d322fd435"
+        },
+        "sources.json"
+      ],
+      "build-commands" : [
+        "cargo --offline fetch --manifest-path Cargo.toml --verbose",
+        "cargo --offline build --release --verbose",
+        "install -Dm755 ./target/release/eternal_mod_manager -t /app/bin/",
+        "install -Dm644 resources/io.github.brunoanc.eternalmodmanager.appdata.xml /app/share/metainfo/io.github.brunoanc.eternalmodmanager.appdata.xml",
+        "install -Dm644 resources/io.github.brunoanc.eternalmodmanager.png /app/share/icons/hicolor/256x256/apps/io.github.brunoanc.eternalmodmanager.png",
+        "install -Dm644 resources/io.github.brunoanc.eternalmodmanager.desktop /app/share/applications/io.github.brunoanc.eternalmodmanager.desktop",
+        "install -Dm644 LICENSE /app/share/licenses/io.github.brunoanc.eternalmodmanager/LICENSE"
+      ]
+    },
+    {
+      "name" : "alacritty",
+      "buildsystem" : "simple",
+      "sources" : [
+        {
+          "type" : "file",
+          "url" : "http://launchpadlibrarian.net/726178572/alacritty_0.13.2-1ubuntu1_amd64.deb",
+          "sha256": "ff20832c7a0d6e7492ce72f8146406df562dfdae389a64251932503df3d24008",
+          "dest-filename": "alacritty.deb",
+          "only-arches": [
+            "x86_64"
+          ]
+        },
+        {
+          "type" : "file",
+          "url" : "http://launchpadlibrarian.net/726178817/alacritty_0.13.2-1ubuntu1_arm64.deb",
+          "sha256": "9c0e00344b18b617cdad1ff6a512a83e93304a5963f780e521a02f6b69298cc4",
+          "dest-filename": "alacritty.deb",
+          "only-arches": [
+            "aarch64"
+          ]
+        }
+      ],
+      "build-commands" : [
+        "ar x alacritty.deb",
+        "tar xf data.tar.zst",
+        "install -Dm755 ./usr/bin/alacritty -t /app/bin/"
```

Comment:
why isn't this built from source?

Reply:
Also why do you need a terminal in here?